.travis.yml

```diff
@@ -95,17 +90,30 @@ x_base_steps:
     services:
       - docker
 
+stages:
+  - check
+  - test
+  - name: deploy
+    if: type != pull_request AND ( tag IS present OR branch = master OR commit_message =~ /\[travis deploy\]/ )
+  - name: deploy-manifest-multiarch
```

Comment:
Why do we break these deploys into two separate stages?

Reply:
Not sure if there's a good reason other than images are built in `deploy` and pushed in both `deploy` and `deploy-manifest-multiarch`.

Reply:
The actual arch-specific images are built and pushed in `deploy`. IIRC these need to exist before we can create and push the multi-arch manifest list.